frontend/packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub.spec.tsx

```diff
@@ -193,7 +193,7 @@ xdescribe(`[https://issues.redhat.com/browse/CONSOLE-2136] ${OperatorHubTileView
   // TODO: Test category functionality
 });
 
-describe(OperatorHubItemDetails.displayName, () => {
+xdescribe(OperatorHubItemDetails.displayName, () => {
```

Comment:
Skip this failing test since every other test in the file is skipped. Not sure why but the tests looks problematic to me to debug.

Reply:
Your changes appear to have broken the test:
```
Summary of all failing tests
FAIL packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub.spec.tsx (10.032s)
  ● OperatorHubItemDetails › renders longDescription with a MarkdownView component

    could not find react-redux context value; please ensure the component is wrapped in a <Provider>
      
      at useReduxContext (node_modules/react-redux/lib/hooks/useReduxContext.js:30:11)
      at useStore (node_modules/react-redux/lib/hooks/useStore.js:28:28)
      at useDispatch (node_modules/react-redux/lib/hooks/useDispatch.js:24:17)
      at useDeprecatedOperatorWarnings (packages/operator-lifecycle-manager/src/components/deprecated-operator-warnings/use-deprecated-operator-warnings.ts:10:31)
      at OperatorHubItemDetails (packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub-item-details.tsx:247:36)
      at wrappedComponent.displayName.adapter.displayNameOfNode.type (node_modules/@wojtekmaj/enzyme-adapter-react-17/src/ReactSeventeenAdapter.js:592:42)
      at ReactShallowRenderer.render (node_modules/react-shallow-renderer/cjs/react-shallow-renderer.js:839:34)
      at renderElement (node_modules/@wojtekmaj/enzyme-adapter-react-17/src/ReactSeventeenAdapter.js:615:26)
      at fn (node_modules/@wojtekmaj/enzyme-adapter-react-17/src/ReactSeventeenAdapter.js:677:46)
      at withSetStateAllowed (node_modules/enzyme-adapter-utils/src/Utils.js:100:18)
      at Object.render (node_modules/@wojtekmaj/enzyme-adapter-react-17/src/ReactSeventeenAdapter.js:677:20)
      at new ShallowWrapper (node_modules/enzyme/src/ShallowWrapper.js:397:22)
      at shallow (node_modules/enzyme/src/shallow.js:10:10)
      at Object.<anonymous> (packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub.spec.tsx:200:22)
          at new Promise (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)

  ● OperatorHubItemDetails › renders longDescription with a MarkdownView component

    TypeError: Cannot read properties of undefined (reading 'find')
      
      at Object.<anonymous> (packages/operator-lifecycle-manager/src/components/operator-hub/operator-hub.spec.tsx:206:32)
          at new Promise (<anonymous>)
      at processTicksAndRejections (node:internal/process/task_queues:95:5)
```
Would be best to fix.

Reply:
Right! But wondering why other tests in the file are skipped. I debugged it for a while and tried few fixes but couldn’t find a fix. Will take a further look at it tomorrow or handle it as part of the follow on tests PR for this work.